config/project.sh

```diff
@@ -4,12 +4,13 @@ export PROJECT_VERSION="0.0.1"
 
 export GOLANG_VERSION="1.9.2"
 
-export API_URL="dev.api.daniel-ortega.mx"
+export API_URL="dev.api.culturacolectiva.com"
```

Comment:
This only work for this domain?

What about if this `gocker` tool is needed in several micro services?

Reply:
yes, at this moment that configuration only works manually, but I've been working to improve this, I think the best way to solve this is through the init interface, but it is not finished yet :(